reactor-netty-http/src/main/java/reactor/netty/http/server/logging/AccessLogArgProviderH2.java

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright (c) 2011-Present VMware, Inc. or its affiliates, All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package reactor.netty.http.server.logging;
+
+import io.netty.channel.socket.SocketChannel;
+import io.netty.handler.codec.http2.Http2HeadersFrame;
+
+/**
+ * @author limaoning
+ */
+final class AccessLogArgProviderH2 extends AbstractAccessLogArgProvider {
+
+	static final String H2_PROTOCOL_NAME = "HTTP/2.0";
+
+	Http2HeadersFrame requestHeaders;
+	Http2HeadersFrame responseHeaders;
```

Comment:
Now the response is only for status, do you mean to replace responseHeaders(Http2HeadersFrame responseHeaders) with status(CharSequence status)?

Reply:
> For the moment do we need the response for something else than status?

Now the response is only for status, do you mean to replace responseHeaders(Http2HeadersFrame responseHeaders) with status(CharSequence status)?

Reply:
The class is internal so if we need the response (for example for response headers) we can always change it, so let's now just set the status, otherwise we need NULL check when accessing the response

Reply:
Ok. And I am thinking initialize the accessLogArgProvider only on the first time the request is received, clear the fields in accessLogArgProvider (e.g. requestHeaders) after log,
`accessLog .apply(accessLogArgProvider).log();
 accessLogArgProvider.clear();`
so can reuse the accessLogArgProvider and help GC. Do you think is that ok?

Reply:
this should be ok